### PR TITLE
Comment out ellipsis in code blocks

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Ellipsis in markdown code blocks are syntax errors.

In order to have smooth transition to Prettier and ESLint the PR comments out such bare ellipses. Refer the [discussion](https://github.com/mdn/content/pull/18171#issuecomment-1179670484) for more context.

### CODE REVIEW

File: files/en-us/web/api/event/bubbles/index.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ A boolean value, which is `true` if the event bubbles up through the DOM tree.
 
 ```js
 function handleInput(e) {
-  // Checks whether the event bubbles and…
+  // Check whether the event bubbles
```
Comment (teoli2003) [inline files/en-us/web/api/event/bubbles/index.md:25]:
```suggestion
  // Check whether the event bubbles passes the event along
```

File: files/en-us/web/api/event/bubbles/index.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ A boolean value, which is `true` if the event bubbles up through the DOM tree.
...
   if (!e.bubbles) {
-    // ... passes the event along if does not
+    // Passes the event along
```
Comment (teoli2003) [inline files/en-us/web/api/event/bubbles/index.md:27]:
```suggestion
```

File: files/en-us/web/api/ext_disjoint_timer_query/getqueryobjectext/index.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ const ext = gl.getExtension('EXT_disjoint_timer_query');
 const query = ext.createQueryEXT();
 ext.beginQueryEXT(ext.TIME_ELAPSED_EXT, query);
 
-// ... drawing ...
+// drawing…
```
Comment (teoli2003) [inline files/en-us/web/api/ext_disjoint_timer_query/getqueryobjectext/index.md:50]:
```suggestion
// Draw
```

File: files/en-us/web/api/filereader/abort_event/index.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ _Also inherits properties from its parent {{domxref("Event")}}_.
                accept="image/png, image/jpeg">
     </div>
 
-    <img src="" class="preview" height="200" alt="Image preview...">
+    <img src="" class="preview" height="200" alt="Image preview…">
```
Comment (teoli2003) [inline files/en-us/web/api/filereader/abort_event/index.md:64]:
```suggestion
    <img src="" class="preview" height="200" alt="Image preview">
```

File: files/en-us/web/api/file_api/using_files_from_web_applications/index.md [hunk 5]
```diff
@@ -480,7 +480,7 @@ if (isset($_FILES['myFile'])) {
 </head>
 <body>
     <div>
-        <div id="dropzone" style="margin:30px; width:500px; height:300px; border:1px dotted grey;">Drag & drop your file here...</div>
+        <div id="dropzone" style="margin:30px; width:500px; height:300px; border:1px dotted grey;">Drag & drop your file here…</div>
```
Comment (teoli2003) [inline files/en-us/web/api/file_api/using_files_from_web_applications/index.md:483]:
```suggestion
        <div id="dropzone" style="margin:30px; width:500px; height:300px; border:1px dotted grey;">Drag & drop your file here</div>
```

File: files/en-us/web/api/filereader/load_event/index.md [hunk 6]
```diff
@@ -59,7 +59,7 @@ _Also inherits properties from its parent {{domxref("Event")}}_.
                accept="image/png, image/jpeg">
     </div>
 
-    <img src="" class="preview" height="200" alt="Image preview...">
+    <img src="" class="preview" height="200" alt="Image preview…">
```
Comment (teoli2003) [inline files/en-us/web/api/filereader/load_event/index.md:62]:
```suggestion
    <img src="" class="preview" height="200" alt="Image preview">
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few nits and we can merge this.

**Comment by OnkarRuikar** [comment]
Some pages on mdn/content have placeholder sections like:
````
…

## Description

## Examples

```js
```
````

Should we remove them?

**Comment by teoli2003** [comment]
> Should we remove them?

Yes, please. That way, when we start enforcing the presence of the mandatory sections, they will be listed as missing the examples. Also, it gives false expectations to the reader that believes there is an example and is disappointed when it reaches the section.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**